main/design_documents/ARM_NEON_SVE_BRIDGE.md

```diff
@@ -0,0 +1,69 @@
+# Design Document for ARM_NEON_SVE_BRIDGE
+
+This document describes the design of the intrinsics defined in
+`NEON_SVE Bridge <../acle.md#neon-sve-bridge>`_. These C intrinsics enable the
```

Comment:
I this this link should become:

```suggestion
[NEON_SVE Bridge](../acle.md#neon-sve-bridge). These C intrinsics enable the
```

(you can see that the link is not rendered here: https://github.com/MDevereau/acle/blob/next-release/main/design_documents/ARM_NEON_SVE_BRIDGE.md)